semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/bad1/error-in-color.txt

```diff
@@ -1,4 +1,4 @@
-[31msemgrep error[0m: Invalid rule schema
+[31m[31msemgrep error[0m: Invalid rule schema
```

Comment:
Should we have `semgrep error` be colored something that isn't red since the entire error is red now?

Reply:
It's actually not, because of the reset characters within the error body.